### PR TITLE
adding summary info to bring into compliance

### PR DESCRIPTION
**Pull request by craigcaseyMSFT**
For compliance, adding summaries that were missing for .NET framework 1.1

### CODE REVIEW

**Comment by craigcaseyMSFT** [comment]
Hi, @gewarren  - I've got another bunch of these ready to review (added missing summaries). Please let me know if anything needs to be updated. Thank you!

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
         <ReturnType>System.Xml.Serialization.XmlAttributeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the UnknownAttribute event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:61]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownAttribute" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 2]
```diff
@@ -127,7 +127,7 @@
         <ReturnType>System.Xml.Serialization.XmlElementEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the UnknownElement event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:130]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownElement" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 3]
```diff
@@ -196,7 +196,7 @@
         <ReturnType>System.Xml.Serialization.XmlNodeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the UnknownNode event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:199]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownNode" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 4]
```diff
@@ -265,7 +265,7 @@
         <ReturnType>System.Xml.Serialization.UnreferencedObjectEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the UnreferencedObject event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:268]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="UnreferencedObject" /> event.</summary>
```

**Comment by craigcaseyMSFT** [comment]
Hi, @gewarren  - made the changes as requested. Please let me know if there are any other changes needed. Thanks much!

File: xml/System.Data.Odbc/OdbcCommand.xml [hunk 5]
```diff
@@ -638,7 +638,7 @@ SELECT * FROM Customers WHERE CustomerID = ?
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>When overridden in a derived class, creates a new instance of a DbParameter object.</summary>
+        <summary>When overridden in a derived class, creates a new instance of a <see cref="DbParameter" /> object.</summary>
```
Comment (gewarren) [inline xml/System.Data.Odbc/OdbcCommand.xml:641]:
```suggestion
        <summary>When overridden in a derived class, creates a new instance of a <see cref="T:System.Data.Common.DbParameter" /> object.</summary>
```

File: xml/System.Security.Cryptography/MD5CryptoServiceProvider.xml [hunk 6]
```diff
@@ -183,7 +183,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Releases the resources that are being used by MD5CryptoServiceProvider</summary>
+        <summary>Releases the resources that are being used by the <see cref="MD5CryptoServiceProvider" />.</summary>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/MD5CryptoServiceProvider.xml:186]:
```suggestion
        <summary>Releases the resources that are being used by the <see cref="T:System.Security.Cryptography.MD5CryptoServiceProvider" />.</summary>
```

File: xml/System.Web.Services.Description/MessageBinding.xml [hunk 7]
```diff
@@ -90,7 +90,7 @@
         <ReturnType>System.Web.Services.Description.ServiceDescriptionFormatExtensionCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the ServiceDescriptionFormatExtensionCollection associated with the DocumentableItem.</summary>
+        <summary>Gets the <see cref="ServiceDescriptionFormatExtensionCollection" /> associated with the <see cref="DocumentableItem" />.</summary>
```
Comment (gewarren) [inline xml/System.Web.Services.Description/MessageBinding.xml:93]:
```suggestion
        <summary>Gets the <see cref="T:System.Web.Services.Description.ServiceDescriptionFormatExtensionCollection" /> associated with the <see cref="DocumentableItem" />.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 8]
```diff
@@ -58,7 +58,7 @@
         <ReturnType>System.Xml.Serialization.XmlAttributeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an object that represents the method that handles the UnknownAttribute event.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownAttribute" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:61]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="T:System.Xml.Serialization.UnknownAttribute" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 9]
```diff
@@ -127,7 +127,7 @@
         <ReturnType>System.Xml.Serialization.XmlElementEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an object that represents the method that handles the UnknownElement event.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownElement" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:130]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="T:System.Xml.Serialization.UnknownElement" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 10]
```diff
@@ -196,7 +196,7 @@
         <ReturnType>System.Xml.Serialization.XmlNodeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an object that represents the method that handles the UnknownNode event.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="UnknownNode" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:199]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="T:System.Xml.Serialization.UnknownNode" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 11]
```diff
@@ -265,7 +265,7 @@
         <ReturnType>System.Xml.Serialization.UnreferencedObjectEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an object that represents the method that handles the UnreferencedObject event.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="UnreferencedObject" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:268]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.UnreferencedObject" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 12]
```diff
@@ -58,7 +58,7 @@
         <ReturnType>System.Xml.Serialization.XmlAttributeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.UnknownAttribute" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:61]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.XmlSerializer.UnknownAttribute" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 13]
```diff
@@ -127,7 +127,7 @@
         <ReturnType>System.Xml.Serialization.XmlElementEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.UnknownElement" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:130]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.XmlSerializer.UnknownElement" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 14]
```diff
@@ -196,7 +196,7 @@
         <ReturnType>System.Xml.Serialization.XmlNodeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.UnknownNode" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:199]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.XmlSerializer.UnknownNode" /> event.</summary>
```

File: xml/System.Xml.Serialization/XmlDeserializationEvents.xml [hunk 15]
```diff
@@ -265,7 +265,7 @@
         <ReturnType>System.Xml.Serialization.UnreferencedObjectEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.UnreferencedObject" /> event.</summary>
```
Comment (gewarren) [inline xml/System.Xml.Serialization/XmlDeserializationEvents.xml:268]:
```suggestion
        <summary>Gets or sets an object that represents the method that handles the <see cref="E:System.Xml.Serialization.XmlSerializer.UnreferencedObject" /> event.</summary>
```

**Pull request merged**